caiman/base/movies.py

```diff
@@ -1604,6 +1604,22 @@ def rgb2gray(rgb):
         elif extension == '.sima':
             raise Exception("movies.py:load(): FATAL: sima support was removed in 1.9.8")
 
+        elif extension == '.isxd':
+            import isx
```

Comment:
Hi, isn't the inscopix API package not open source? Also, from my experience of using isx ~1 year ago, it imports a bunch of libs (even for just using the io API) which can cause issues with the local Qt in the environment.

Reply:
Basically, if `pyqt` is already imported in the current python process, import isx causes the kernel to crash because the Qt event loop has already been started, but the isx import tries to start another even loop.

Reply:
Hi @kushalkolar, yes absolutely, sorry about that I didn't mean to push on this PR! Reverting this now ;)